components/cron-pvc/cron-job/job.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cleanup-pvc-pipelinerun
+spec:
+  # Repeat each 5 min
+  schedule: "*/5 * * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  suspend: false
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: pvc-cleaner
+          containers:
+          - name: cleanup-pvc-pipelinerun
+            image: registry.redhat.io/openshift4/ose-cli
+            imagePullPolicy: IfNotPresent
+            command:
+            - /bin/sh
+            - -c
+            - |
+              echo "Clean up pvc subpaths..."
+
+              oc apply -f - <<EOF
+              apiVersion: tekton.dev/v1beta1
+              kind: TaskRun
+              metadata:
+                name: cleanup-build-directories-run
+              spec:
+                serviceAccountName: pipeline
+                taskRef:
+                  name: cleanup-build-directories
+                  bundle: quay.io/redhat-appstudio/appstudio-tasks:cleanup-build-directories-v0.1.4
+                timeout: 1h0m0s
+                workspaces:
+                  - name: source
+                    persistentVolumeClaim:
+                      claimName: app-studio-default-workspace
+              EOF
+
+              oc wait --for=condition=succeeded taskrun cleanup-build-directories-run --timeout=120s
+              oc delete taskrun cleanup-build-directories-run
```

Comment:
do not delete the taskrun, then you will see that it's removing subpaths for living pipelines...

Reply:
```
step-cleanup-build-directories
App Studio Utility Task cleanup-build-directories
Removing Unused Build Directories
Cleanup:
Directory: ./pv-devfile-build
Directory: ./pv-docker-build
Directory: ./pv-nodejs-builder
Valid: pv-devfile-build-2022-02-11-134629
Valid: pv-devfile-build-2022-02-11-134634
Valid: pv-devfile-build-2022-02-11-134640
Valid: pv-devfile-build-2022-02-11-134646
Valid: pv-devfile-build-2022-02-11-140303
Valid: pv-devfile-build-2022-02-11-140309
Valid: pv-devfile-build-2022-02-11-140314
Valid: pv-devfile-build-2022-02-11-140320
Valid: pv-docker-build-2022-02-11-134652
Valid: pv-docker-build-2022-02-11-140326
Valid: pv-java-builder-2022-02-11-134703
Valid: pv-java-builder-2022-02-11-140336
Valid: pv-nodejs-builder-2022-02-11-134658
Valid: pv-nodejs-builder-2022-02-11-140332
Removing: 572K	./pv-devfile-build
Removing: 268K	./pv-docker-build
Removing: 12K	./pv-nodejs-builder
cleanup-build-directories.sh Before: 860K	. After: 20K	.
```
The cleanup script does not count with subfolders.